<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoanDialog</class>
 <widget class="QWidget" name="LoanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableWidget::item::selected{
	background-color: rgb(114, 166, 162);
}

QTableWidget::item:selected:active {
    background: #e5f3ff;
	color: black;
}

QTableWidget::item:hover {
    background:  #e5f3ff;
}

QLineEdit {
border: 2px solid #8f8f91;
border-radius: 6px;
    background-image: url(:/icons/search.png);
    background-repeat: no-repeat;
    background-position: left;
    color: #252424;
    font-family: SegoeUI;
    font-size: 12px;
    padding: 0 0 0 20; /* left padding (last number) must be more than the icon's width */
}
</string>
  </property>
  <widget class="QLineEdit" name="searchBook">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Search book by ISBN</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableBook">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>281</width>
     <height>301</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ISBN</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="searchUser">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Search user by name</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableUser">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>281</width>
     <height>301</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="confirmBttn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>540</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;
	border-radius: 8px;
	background-color: rgb(131, 131, 131);
    border-style: outset;
    border-width: 2px;
    border-color:rgb(115, 115, 115);
    padding: 6px;
}
QPushButton:pressed {
    border-style: inset;
}

QPushButton:hover {
	background-color: rgb(115, 115, 115);
    color:white;
    border-style: outset;
}</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="subtext">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>661</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: grey;</string>
   </property>
   <property name="text">
    <string>Search and select the book and the subscriber</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: grey;</string>
   </property>
   <property name="text">
    <string>Make a new loan</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>450</y>
     <width>221</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="term">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Term of the loan</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="termSpinBox">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="value">
       <number>14</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/bg_book.jpg);</string>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>searchBook</zorder>
  <zorder>tableBook</zorder>
  <zorder>searchUser</zorder>
  <zorder>tableUser</zorder>
  <zorder>confirmBttn</zorder>
  <zorder>subtext</zorder>
  <zorder>title</zorder>
  <zorder>verticalLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
